src/status_im/config.cljs

```diff
@@ -150,3 +150,6 @@
 (goog-define STATUS_BACKEND_SERVER_ROOT_DATA_DIR "")
 ;; if you're using android simulator, I suggest set the env variable to "http://10.0.2.2:"
 (goog-define STATUS_BACKEND_SERVER_IMAGE_SERVER_URI_PREFIX "https://localhost:")
+
+(def re-frisk-host (get-config :RE_FRISK_HOST "http://localhost"))
```

Comment:
This is the default, but what I like to do is change this to my computer's local hostname. For example, when the mobile device and my laptop are on the same network, my phone can access my computer by its local hostname like `dev.local`.

This is nice because using the local hostname avoids needing to update the `RE_FRISK_HOST` variable whenever the IP address changes.